types/react-transition-group/react-transition-group-tests.tsx

```diff
@@ -17,6 +17,7 @@ const Test: React.StatelessComponent = () => {
         <TransitionGroup
```

Comment:
@RyanCavanaugh The module is converted from ES6 `default` export to `module.export = exports['default']` in the compiled code. I'm using the `namespace` hack here, also mentioned in your Stackoverflow post to be eble to export corresponding types along with the classes themselves. I'm not sure if there is any other way to export smething else than the main class if just using `module.export = SomeClass`. I want to be able to import all the types along with cjs import ie.:
```ts
import TransitionGroup = require("react-transition-group/TransitionGroup");
import { TransitionGroupProps } from "react-transiton-group/TransitionGroup";
```
I don't think it woul be possible with `require()` syntax or am I missing something?